doc/previewflag.md

```diff
@@ -0,0 +1,18 @@
+# ASAB WebUI PreviewFlag component
```

Comment:
@queuing4oranges Here it is more about `setFlag` method than the component itself. Please update accordingly (also the filename)